app/javascript/src/components/Dashboard/Contacts/Table/TableColumn.js

```diff
@@ -0,0 +1,3 @@
+import { buildContactsTableColumnData } from "./utils";
+
+export const CONTACTS_TABLE_COLUMNS = buildContactsTableColumnData();
```

Comment:
You are not paying attention to the filenames. Go through the Slack thread.

Reply:
changed the file name to column.js and pushed the changes.